charts/edge/values.yaml

```diff
@@ -78,3 +78,6 @@ pvcName: ""
 
 maxDiskBufferBytes: 0
 maxMemBufferEvents: 0
+
+reduceByteThresholdPerState: 0
```

Comment:
I think the internal defaults are too low for almost any edge deployment. What would you think of setting these values higher (100MiB, 1GiB) and taking away the conditional templating in `statefulset.yaml`?

Reply:
Based on the [ticket](https://mezmo.atlassian.net/browse/LOG-19194) we're asked to set the limits to 10Mb and 100Mb, maybe lets set these values as default?

Reply:
That was for multitenant SaaS, right? I think we should default higher for this application.

Reply:
Ok, we can do that

Reply:
Increased to 100Mb and 1Gb